OpalImagePickerTests/OpalImagePickerRootViewControllerTest.swift

```diff
@@ -36,13 +36,23 @@ class OpalImagePickerRootViewControllerTest: XCTestCase {
     
     func testGetCollectionViewLayout() {
         let imagePicker = OpalImagePickerRootViewController()
-        imagePicker.loadViewIfNeeded()
+        if #available(iOS 9.0, *) {
+            imagePicker.loadViewIfNeeded()
+        } else {
+            // Fallback on earlier versions
+            imagePicker.loadView()
```

Comment:
It would be simpler to just change this to `imagePicker.loadView()` since it is just a unit test.